tests/phpunit/test-cache.php

```diff
@@ -114,10 +143,15 @@ public function test_redis_reload_force_cache_flush() {
 		// Force a bad connection
 		$redis_server['host'] = '127.0.0.1';
 		$redis_server['port'] = 9999;
-		$this->cache->redis->connect( $redis_server['host'], $redis_server['port'], 1, null, 100 );
+		$this->cache->build_client_connection( $this->cache->build_client_parameters( $redis_server ) );
 		// Setting cache value when redis connection fails saves wakeup flush
 		$this->cache->set( 'foo', 'bar' );
-		$this->assertEquals( 'WP Redis: Redis server went away', $this->cache->last_triggered_error );
+		$this->assertTrue(
+			$this->cache->message_matches(
+				str_replace( 'WP Redis: ', '', $this->cache->last_triggered_error ),
+				$this->cache->retry_exception_messages()
+			)
+		);
```

Comment:
Because Predis has different error messages than PHPRedis, I opted to verify that the message was in the list of acceptable exception messages. Does that work for you?